src/editorial/web/package.json

```diff
@@ -25,6 +25,9 @@
     "verbump:preminor": "yarn version --preminor --preid rc --no-git-tag-version",
     "verbump:prerelease": "yarn version --prerelease --preid rc --no-git-tag-version"
   },
+  "dependencies": {
+    "@guardian/types": "^6.0.0"
+  },
```

Comment:
do we want to bundle types? or should we use `peerDependencies` instead?

Reply:
Good question. Expecting platforms to use types seems very fair. @SiAdcock , what think you?

Reply:
Hmm. Would it not be a `devDependency`? Types are a compile-time concern rather than a runtime one.